app/views/state_file/questions/data_transfer_offboarding/edit.html.erb

```diff
@@ -13,13 +13,12 @@
     <p><%= vita_introduction %></p>
   <% end %>
 
-  <% vita_link = I18n.locale == :en ? StateFile::StateInformationService.vita_link_en(current_state_code) : StateFile::StateInformationService.vita_link_es(current_state_code) %>
-  <% if vita_link.present? %>
-    <p><%= link_to t("state_file.questions.eligible.faq_link"), state_faq_path(us_state: current_state_code) %></p>
-    <%= link_to(vita_link, class: "button button--primary button--wide spacing-below-15", target: "_blank", rel: "noopener noreferrer") do %>
-      <%= t('state_file.questions.eligible.vita_option.connect_to_vita') %>
-    <% end %>
-  <% end %>
+  <% # i18n-tasks-use t('state_file.questions.eligible.vita_option.connect_to_vita.nj') # hint for the i18n linter that, yes, we are using this key (sometimes) %>
```

Comment:
@mluedke2 is `i18n-tasks-use` a command? Sorry I'm confused about this comment, could you help me understand?

Reply:
it's a linter pattern i copied from elsewhere in the app (if you text search for the comment you will see many of them throughout). without it, the dynamic key will fail the automated i18n health checks because it won't know the key was used.

Reply:
Oooooh I see right because there will be a failure in the i18n health checkup if we are not using a key. I hadn't realized we had moved away from adding it to the ignore file to mentioning it as a comment but you're right I do see it elsewhere in the app.